apps/finance/app/src/components/useDownloadData.js

```diff
@@ -0,0 +1,94 @@
+import { useContext, useCallback } from 'react'
+import { format } from 'date-fns'
+import { saveAs } from 'file-saver'
+import { useCurrentApp } from '@aragon/api-react'
+import { IdentityContext } from './IdentityManager/IdentityManager'
+import { toChecksumAddress } from '../lib/web3-utils'
```

Comment:
Nitpick: let's alphabetize :).

Reply:
You mean alphabetize by import names? 😃 AFAIK by the rules we have, this is alphabetized:
- npm packages first and alphabetized
- namespaced packages after npm packages, also alphabetized
- local files, alphabetized

Reply:
The lib imports are currently upside down :)

Reply:
Aaah gotcha!